### PR TITLE
add program_161+ exercises

### PR DESCRIPTION
**Pull request by nngrz**

### CODE REVIEW

**Review by balinskia — APPROVED** [review]

**Comment by balinskia** [comment]
If you already have program_166 folder, then there is no need to create inside this folder files named program_166.html and program_166.js etc. Better to name them index.html and index.js

**Comment by balinskia** [comment]
Outdated PR title. If you are adding many exercises you can just write "Adding program_161+ exercises"

PR title doesn't match standard. I.e "add "program161 and program162", not "program_161 and program162"

File: program_166/program_166.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let hands = ["rock", "paper", "scissor"]
+
+// Create a function that returns a random item from the array
+function getRandomItem() {
+    let randomNumber = Math.floor(Math.random() * 3)
+    // return randomNumber for debug
```
Comment (balinskia) [inline program_166/program_166.js]: What exactly you are dubbiging here?

**Review by balinskia — CHANGES_REQUESTED** [review]

File: program_166/program_166.js (hunk 1 above)
Comment (nngrz) [reply program_166/program_166.js]: check if get code for get random index correct

Comment (balinskia) [reply program_166/program_166.js]: Yeah, it is. I just don't understand this comment (just delete it)

**Comment by balinskia** [comment]
> Outdated PR title. If you are adding many exercises you can just write "Adding program_161+ exercises"

Bump

**Comment by balinskia** [comment]
> If you already have program_166 folder, then there is no need to create inside this folder files named program_166.html and program_166.js etc. Better to name them index.html and index.js

Please do it for all files

**Review by balinskia — CHANGES_REQUESTED** [review]
See comments

File: program_169/index.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body {
+    margin: 0;
```
Comment (balinskia) [inline program_169/index.css]: Rename to match the standard

**Review by balinskia — CHANGES_REQUESTED** [review]

**Review by balinskia — APPROVED** [review]

File: program_169/program_169.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+const characters = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z","a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9","~","`","!","@","#","$","%","^","&","*","(",")","_","-","+","=","{","[","}","]",",","|",":",";","<",">",".","?",
+"/"];
```
Comment (balinskia) [inline program_169/program_169.js:2]: I think this is not finished?

**Review by balinskia — CHANGES_REQUESTED** [review]

File: program_169/program_169.js (hunk 3 above)
Comment (nngrz) [reply program_169/program_169.js:2]: > I think this is not finished?
not finished, I pushed by accident

**Pull request merged**